src/inmanta/server/extensions.py

```diff
@@ -103,18 +106,19 @@ class FeatureManager:
 
         slices:
             slice_name:
-                feature_name: bool
+                feature_name: FeatureValueTypes
```

Comment:
You need to search for features. We define 2 types at the moment in this file:

```
class BoolFeature(Feature[bool]):
    """A feature that is on or off. When no value is given it is enabled."""

    def __init__(self, slice: str, name: str, description: str = "") -> None:
        super().__init__(slice, name, description, True)


class StringListFeature(Feature[list[str]]):
    """A feature that holds a list of allowed values. When the list contains "*" it matches everything."""

    def __init__(self, slice: str, name: str, description: str = "") -> None:
        super().__init__(slice, name, description, default_value=["*"])
```

For completeness I decided to also include `str` but that is strictly not required at this point.

Reply:
The reason why I had to change it, is that the previous definition was to complex and caused import issues that I would not easily solve without making things even more complex.

Reply:
Since this is untyped I would expect it to be plain `object`, but I have no strict objection against the union. I would have assumed that the features/manager would do some sort of validation but looks like it doesn't, so the whole typing is a bit of a lie there.